ssz-rs/src/de.rs

```diff
@@ -2,7 +2,7 @@ use crate::{
     error::{InstanceError, TypeError},
     lib::*,
     ser::BYTES_PER_LENGTH_OFFSET,
-    SimpleSerialize,
+    Sized as SSZSized,
```

Comment:
I think we can just import `Sized` here

if you think that is confusing w/ the `std::Sized` then I have a slight preference for using `crate::Sized` down below instead of this `SSZSized` symbol